File: mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int height;
    do
    {
        height = get_int("Choose an height for the pyramid between 1 and 8: ");
    }
    while (height < 1 || height > 8);
    int counter = 1;
    for (int i = 0; i < height; i++)
    {
        for (int j = height - i - 1; j > 0; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < counter; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < counter; k++)
        {
            printf("#");
        }
        printf("\n");
        counter++;
    }
}

File: sentimental-mario-more/mario.py
import cs50

while True:
    height = cs50.get_int("Choose an height for the pyramid between 1 and 8: ")
    if height >= 1 and height <= 8:
        break

for i in range(height):
    print(" " * (height -i -1), end='')
    print("#" * (i+1), end='')
    print("  ", end='')
    print("#" * (i+1), end='')
    print("")
